CONTRIBUTING.rst

```diff
@@ -210,16 +210,9 @@ developers and contributors can then review your code and offer suggestions.
 
 Release procedure
 =================
+Ultraplot follows EffVer (Effectual Versioning). Changes to the version number ``X.Y.Z`` will reflect the effect on users: the major version ``X`` will be incremented for changes that require user attention (like breaking changes), the minor version ``Y`` will be incremented for safe feature additions, and the patch number ``Z ` will be incremented for changes users can safely ignore.
```

Comment:
```suggestion
Ultraplot follows EffVer (Effectual Versioning). Changes to the version number ``X.Y.Z`` will reflect the effect on users: the major version ``X`` will be incremented for changes that require user attention (like breaking changes), the minor version ``Y`` will be incremented for safe feature additions, and the patch number ``Z`` will be incremented for changes users can safely ignore.
```

Reply:
We should link to the EffVer website. We should also update the language of this to reflect the language used by effver.

Reply:
done and done